packages/metro/src/DeltaBundler/DeltaCalculator.js

```diff
@@ -180,9 +188,16 @@ class DeltaCalculator<T> extends EventEmitter {
     if (type === 'delete') {
       this._deletedFiles.add(filePath);
       this._modifiedFiles.delete(filePath);
-    } else {
+      this._addedFiles.delete(filePath);
+    } else if (type === 'add') {
```

Comment:
Are there any tests covering `'add'` events? I don't see any in `DeltaCalculator-test.js`.

Reply:
Seems like [this tests](https://github.com/facebook/metro/blob/419caf75656c373cdb832a45297d40d0c1b19cfb/packages/metro/src/DeltaBundler/__tests__/DeltaCalculator-test.js#L268) adding files already.